src/style/main.less

```diff
@@ -52,35 +52,35 @@ input {
 
 @font-face {
   font-family: 'Adobe Clean';
-  src: url('file://shared/font/AdobeClean-Regular.otf');
+  src: url('~file://shared/font/AdobeClean-Regular.otf');
```

Comment:
`~` allows less loader to parse these requires as absolute paths and leave them untouched.